From 47163267a5c4e89ae4c3f31530e968ed65b1ac49 Mon Sep 17 00:00:00 2001
From: Arthur Liberman <arthur_liberman@hotmail.com>
Date: Tue, 20 Mar 2018 02:25:55 +0200
Subject: [PATCH] Add A95X R2 (Abox A1 Max) specific DTBs. The
 'internal_eth_pins' node conflicts with the VFD pins, preventing the VFD
 driver from starting. Built in ethernet still works without this node.

Signed-off-by: Arthur Liberman <arthur_liberman@hotmail.com>
---
 gxl_p281_1g_a95xr2.dts | 9 +++++++++
 gxl_p281_2g_a95xr2.dts | 9 +++++++++
 2 files changed, 18 insertions(+)
 create mode 100644 gxl_p281_1g_a95xr2.dts
 create mode 100644 gxl_p281_2g_a95xr2.dts

diff --git a/gxl_p281_1g_a95xr2.dts b/gxl_p281_1g_a95xr2.dts
new file mode 100644
index 0000000..6c9fba8
--- /dev/null
+++ b/gxl_p281_1g_a95xr2.dts
@@ -0,0 +1,9 @@
+#include "gxl_p281_1g.dts"
+
+/ {
+	le-dt-id = "gxl_p281_1g_a95xr2";
+
+	pinmux {
+		/delete-node/ internal_eth_pins;
+	};
+};
diff --git a/gxl_p281_2g_a95xr2.dts b/gxl_p281_2g_a95xr2.dts
new file mode 100644
index 0000000..a14f5de
--- /dev/null
+++ b/gxl_p281_2g_a95xr2.dts
@@ -0,0 +1,9 @@
+#include "gxl_p281_2g.dts"
+
+/ {
+	le-dt-id = "gxl_p281_2g_a95xr2";
+
+	pinmux {
+		/delete-node/ internal_eth_pins;
+	};
+};
